### PR TITLE
added documentation for translation review

### PR DESCRIPTION
**Pull request by shubhamkorde**
This PR adds documentation for translation reviews. The documentation from the forked repo is hosted here: https://oppia-documentation.readthedocs.io/en/review-translations-documentation/

### CODE REVIEW

**Comment by shubhamkorde** [comment]
@audstav PTAL

File: docs/index.rst
Comment (audstav) [inline docs/index.rst:31]: Please change 'voiceover' to 'voiceovers' here

**Comment by audstav** [comment]
@shubhamkorde Thanks, this looks pretty good! I left a few minor comments for when you get a chance.

@DubeySandeep can you PTAL at this as well, I will leave it to you to approve and merge :)

File: docs/reviewers/review.rst
Comment (sagangwee) [inline docs/reviewers/review.rst]: Nit: For consistency, prefer North American English, i.e. "authorized" rather than "authorised".

File: docs/reviewers/review.rst
Comment (sagangwee) [inline docs/reviewers/review.rst]: Grammar fixes and minor improvements:

```suggestion
The review rights of a user are listed in the header of the Contributor Dashboard under the **My Contributions** tab. Review rights for translations and voiceovers are language specific whereas those for questions are not. The languages in which you can review translations and voiceovers are also listed in the header.
```

File: docs/reviewers/review.rst
Comment (sagangwee) [inline docs/reviewers/review.rst]:
```suggestion
Reviewing Lesson Contributions
```

File: docs/reviewers/translations.rst
Comment (sagangwee) [inline docs/reviewers/translations.rst]:
```suggestion
Oppia's lessons are translated to multiple languages to ensure a wide range of students across the world can learn in their own language. After translations are submitted, they are sent for review. As a reviewer, you can review all the translations in the language for which you have review rights.
```

File: docs/reviewers/translations.rst
Comment (sagangwee) [inline docs/reviewers/translations.rst]:
```suggestion
2. Select the translation you want to review buy clicking on the |reviewbtn| button of that particular translation. A translation review modal will open up as shown in the figure.
```

File: docs/reviewers/translations.rst
Comment (sagangwee) [inline docs/reviewers/translations.rst]:
```suggestion
5. If the translation is correct, click on the |acceptandreviewnextbtn| button to approve it and review the next translation. Adding a review message is not mandatory when accepting a translation.
```

File: docs/reviewers/translations.rst
Comment (sagangwee) [inline docs/reviewers/translations.rst]:
```suggestion
3. The left side text box contains the original text in English and the right side text box contains the translated text. Check if the translation is gramatically correct and conveys the same meaning as the original text. Please follow :ref:`the guidelines <reviewguidelines>`.
```

File: docs/reviewers/translations.rst
Comment (sagangwee) [inline docs/reviewers/translations.rst]:
```suggestion
6. If the translation is not correct, type a review message in the text box with the label 'Review message' explaining the reason for rejection and any possible improvements and suggestions. This message would be visible to the translator. Note that you cannot reject a translation without a review message.  Click on the |rejectandreviewnextbtn| button to continue reviewing other translations.
```

File: docs/reviewers/translations.rst
Comment (sagangwee) [inline docs/reviewers/translations.rst]:
```suggestion
7. Continue the same procedure until you have completed reviewing all the translations.
```

File: docs/reviewers/translations.rst
Comment (sagangwee) [inline docs/reviewers/translations.rst]: Not sure if "persuasiveness" makes sense here?

```suggestion
To ensure that translations convey the same meaning with the same persuasiveness and simplicity as the original text, please keep these principles in mind:
```

File: docs/reviewers/translations.rst
Comment (sagangwee) [inline docs/reviewers/translations.rst]:
```suggestion
 * Ensure that the the voice and meaning of the original text is preserved. It doesn't mean you have to accept only literal translations. If the language is natural, understandable, and stays close to the source, the translation is worth accepting.
```

**Review by sagangwee — CHANGES_REQUESTED** [review]
@shubhamkorde Thanks for the changes. Added some comments.

File: docs/reviewers/review.rst
Comment (shubhamkorde) [reply docs/reviewers/review.rst]: Done!

File: docs/reviewers/review.rst
Comment (shubhamkorde) [reply docs/reviewers/review.rst]: Done!

File: docs/reviewers/translations.rst
Comment (shubhamkorde) [reply docs/reviewers/translations.rst]: Done.

File: docs/reviewers/translations.rst
Comment (shubhamkorde) [reply docs/reviewers/translations.rst]: Done.

File: docs/reviewers/translations.rst
Comment (shubhamkorde) [reply docs/reviewers/translations.rst]: Done.

File: docs/reviewers/translations.rst
Comment (shubhamkorde) [reply docs/reviewers/translations.rst]: Done.

File: docs/reviewers/translations.rst
Comment (shubhamkorde) [reply docs/reviewers/translations.rst]: Done.

File: docs/reviewers/translations.rst
Comment (shubhamkorde) [reply docs/reviewers/translations.rst]: should I replace 'persuasiveness' with 'effectiveness'?

File: docs/reviewers/translations.rst
Comment (shubhamkorde) [reply docs/reviewers/translations.rst]: Done.

File: docs/reviewers/translations.rst
Comment (sagangwee) [inline docs/reviewers/translations.rst]:
```suggestion
To ensure that translations convey the same meaning as the original text, please keep these principles in mind:
```

Let's just remove it.

**Review by sagangwee — APPROVED** [review]
@shubhamkorde Thanks. LGTM, just one last change.

**Review by sagangwee — APPROVED** [review]

**Comment by sagangwee** [comment]
@seanlip Can you merge this, or give me write access to this repo? Thanks.

**Comment by seanlip** [comment]
Done! @sagangwee gave you write access.

**Pull request merged**
